extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authorization/endpoint/BasicAuthorizerResource.java

```diff
@@ -347,24 +505,59 @@ public Response getCachedSerializedUserMap(
       @PathParam("authorizerName") final String authorizerName
   )
   {
-    return resourceHandler.getCachedMaps(authorizerName);
+    return resourceHandler.getCachedUserMaps(authorizerName);
+  }
+
+  /**
+   * @param req HTTP request
+   *
+   * @return serialized groupMapping map
+   */
+  @GET
+  @Path("/db/{authorizerName}/cachedSerializedGroupMappingMap")
+  @Produces(SmileMediaTypes.APPLICATION_JACKSON_SMILE)
+  @Consumes(MediaType.APPLICATION_JSON)
+  @ResourceFilters(BasicSecurityResourceFilter.class)
+  public Response getCachedSerializedGroupMap(
+      @Context HttpServletRequest req,
+      @PathParam("authorizerName") final String authorizerName
+  )
+  {
+    return resourceHandler.getCachedGroupMappingMaps(authorizerName);
   }
 
 
   /**
-   * Listen for update notifications for the auth storage
+   * Listen for update notifications for the user auth storage
    */
   @POST
-  @Path("/listen/{authorizerName}")
+  @Path("/listen/users/{authorizerName}")
```

Comment:
We should keep a copy of this endpoint at `/listen/{authorizerName}` for backwards compatibility

Reply:
Done